addon/index.js

```diff
@@ -3,6 +3,8 @@ import Response from './response';
 import faker from './faker';
 import Model from './orm/model';
 import Serializer from './serializer';
+import ActiveModelSerializer from './serializers/active-model-serializer';
+import JSONAPISerializer from './serializers/json-api-serializer';
```

Comment:
shouldn't it be `JsonApiSerializer` to keep consistency (alternative, we should change that also in other places?)